rsc/application.properties.dist

```diff
@@ -36,6 +36,12 @@ spring.mvc.pathmatch.matching-strategy=ANT_PATH_MATCHER
 ### Security token secret (JWT)
 jwt.token.secret=JWT_TOKEN_SECRET
 
+### JWT token validity, 30 minutes (900,000 milliseconds)
+jwt.token.validityInMilliseconds=900000
+
+### JWT token validity for remember me, 3 days (604,800,000 milliseconds)
+jwt.token.validityInMillisecondsForRememberMe=604800000
+
```

Comment:
LGTM. Just wondering if to ease the configuration for the SA, they could be expressed in seconds (and converted in the code), wdyt?

Reply:
@mwithi Honestly I don't know who is going to fill this properties, so I will trust your point of view on that. If you want I can change this

Reply:
Yes, I think we can use seconds as uom in this file, thanks!